src/controllers/albumController.js

```diff
@@ -4,17 +4,48 @@ const CustomError = require('../errors');
 const logger = require('../../logs/logger');
 
 const createAlbum = async (req, res) => {
+  /*
```

Comment:
swagger-autogen seems to get a bit confused about `req.body.user = req.user.userId` and thinks that that means that the API request actually has a body where the caller passes in a user object. But this is actually coming from our cookie, and the body for this request is all the info for an album. 

We can define the fields expected in a definition object called `NewAlbum` in the swagger.js file. This probably doesn't include all the same fields as the `Album` that's returned (eg: the caller doesn't give us the average rating or number of reviews in the creation request). 

Then here in the controller, we can tell swagger-autogen about this as a body paramater:

```
     #swagger.parameters['newAlbum'] = {
        in: 'body',
        description: 'Album information to use for creation.',
        required: true,
        type: 'object',
        schema: { $ref: '#/definitions/NewAlbum' }
     }
```

Reply:
@akosasante just did it- but can't test it in postman - i log in, create an order and then do GET request to {{MUSICURL}}/albums/64f0777055b548585ca3c18c/listOfUsersWhoPurchasedThisAlbum and get the error that there's no such album
![Captura de pantalla 2023-08-31 a las 13 21 39](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/103755090/f01b30a0-9d5f-4fd1-84c2-c940c6b2bbca)
![Captura de pantalla 2023-08-31 a las 13 21 30](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/103755090/d64db6d9-4141-42d5-bf2c-be6f0b5dc909)